src/main/backend.ts

```diff
@@ -72,7 +72,7 @@ export class MainBackend extends BaseBackend<ElectronOptions> implements CommonB
 
     if (this._isNativeEnabled()) {
       // eslint-disable-next-line @typescript-eslint/no-unused-vars
-      forget(this._installNativeHandler());
+      forget(this._installNativeHandler(options.initialScope));
```

Comment:
The Core SDK already handles this option and writes it into the current hub's scope. It would be great if you just read from that scope, rather than handling the option another time. 

However, this means you'd have to pay attention to the sequence of initialization, because this has to run _after_ the Core SDK has initialized the scope.